assets/blocks/sensei-single-course-blocks.js.js

```diff
@@ -7,6 +7,7 @@ import {
 	CourseOutlineLessonBlock,
```

Comment:
This file name is with a double `.js` extension.

Reply:
Oops d3605de42fe8e583a34ae61137a9ffa6839a9589